library/src/test/java/com/a99/rxplaces/AutocompleteTypeTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.a99.rxplaces
+
+import com.a99.rxplaces.options.AutocompleteType
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+
+class AutocompleteTypeTest {
+  @Test
+  fun test_toString() {
+    // when
+    val cities = AutocompleteType.CITIES.toString()
+    val regions = AutocompleteType.REGIONS.toString()
+    val establishment = AutocompleteType.ESTABLISHMENT.toString()
+    val address = AutocompleteType.ADDRESS.toString()
+    val geocode = AutocompleteType.GEOCODE.toString()
+
+    // then
+    assertThat(cities, `is`("(cities)"))
```

Comment:
Can you please create another test for the case when we are inserting `()` in the parameters?